eng/targets/Imports.targets

```diff
@@ -171,7 +171,7 @@
       Disable analyzers via an MSBuild property settable on the command line.
     -->
     <ItemGroup>
-      <Analyzer Remove="@(Analyzer)" />
+      <Analyzer Remove="@(Analyzer)" Condition="'%(FileName)' != 'CSharpSyntaxGenerator'" />
```

Comment:
❔ Can we use the new property for this?

/cc @mavasani

Reply:
What new property?

Reply:
@jasonmalinowski https://github.com/dotnet/roslyn/pull/46669. We should be able to get rid of `DisableAnalyzers` target and `UseRoslynAnalyzers` property in Roslyn completely in favor of just setting `RunAnalyzersDuringBuild` appropriately.

Reply:
Ah, right forgot that went in. Yeah, we can try just using that now.